packages/reporters/liquidation-reporter/liquidation-reporter.ts

```diff
@@ -233,32 +233,26 @@ export async function fetchUmaEcosystemData() {
         console.error(error);
       }
     }
-    // Finally, remove any elements that have undefined params.
```

Comment:
previously we were editing an element while iterating over it. this is really hard to reason about and debug. I've changed it to a more expressive verbose syntax that makes it easier to see what's happening.